app/components/App.js

```diff
@@ -29,6 +47,13 @@ class App extends Component {
           color="#841584"
           accessibilityLabel="About"
         />
+        <Button
+          onPress={() => this.triggerLogin()}
+          title="Login with Trakt"
+          color="#841584"
```

Comment:
I agree! That's a good idea, let's keep it in mind for when we start building the UI.